openstack/keystone/templates/deployment-api.yaml

```
@@ -41,6 +41,7 @@ spec:
         configmap-bin-hash: {{ include (print $.Template.BasePath "/configmap-bin.yaml") . | sha256sum }}
         secrets-bin-hash: {{ include (print $.Template.BasePath "/secret-bin.yaml") . | sha256sum }}
         secrets-hash: {{ include (print $.Template.BasePath "/secrets.yaml") . | sha256sum }}
+        federation-hash: {{ include (print $.Template.BasePath "/federation.yaml") . | sha256sum }}
```

```suggestion
        {{- if .Values.federation.oidc.enabled }}
        federation-hash: {{ include (print $.Template.BasePath "/federation.yaml") . | sha256sum }}
        {{- end }}
```

Otherwise unrelated changes in the template even when `enabled=false` would be causing API restarts